src/compiler/transformer.ts

```diff
@@ -72,7 +72,7 @@ import {
     transformNodeModule,
     transformSystemModule,
     transformTypeScript,
-    VariableDeclaration
+    VariableDeclaration,

```

Comment:
I think @andrewbranch previously noted this, but we'll have to make sure that organize doesn't screw this up.